tests/integration/common/transaction_utils.rs

```diff
@@ -78,47 +80,163 @@ fn tx_hash_invoke_v0(
     ])
 }
 
+fn tx_hash_declare_v1(sender_address: Felt252, max_fee: Felt252, class_hash: Felt252, nonce: Felt252) -> Felt252 {
```

Comment:
Document these functions.

Reply:
e0e988a -- pretty basic, let me know if there's something specific you want to see documented